src/dex/sundaeswap-v3.ts

```diff
@@ -0,0 +1,238 @@
+import { LiquidityPool } from './models/liquidity-pool';
+import { BaseDataProvider } from '@providers/data/base-data-provider';
+import { Token } from './models/asset';
+import { BaseDex } from './base-dex';
+import {
+    AssetBalance,
+    DatumParameters,
+    DefinitionConstr,
+    DefinitionField,
+    PayToAddress,
+    RequestConfig, SpendUTxO,
+    SwapFee,
+    UTxO
+} from '@app/types';
+import { DefinitionBuilder } from '@app/definition-builder';
+import { correspondingReserves, tokensMatch } from '@app/utils';
+import { AddressType, DatumParameterKey } from '@app/constants';
+import pool from '@dex/definitions/sundaeswap-v1/pool';
+import order from '@dex/definitions/sundaeswap-v1/order';
+import { BaseApi } from '@dex/api/base-api';
+import { Script } from 'lucid-cardano';
+import { SundaeSwapV3Api } from '@dex/api/sundaeswap-v3-api';
+
+export class SundaeSwapV3 extends BaseDex {
+
+    public static readonly identifier: string = 'SundaeSwapV1';
+    public readonly api: BaseApi;
+
+    /**
+     * On-Chain constants.
+     */
+    // TODO - update for mainnet
+    public readonly orderAddress: string = 'addr_test1zpyyj6wexm6gf3zlzs7ez8upvdh7jfgy3cs9qj8wrljp925y2usavetr7qg4zg6cc8kqup4veczmfykgquw2kmmuhs5qkzsen3';
```

Comment:
You can get these all from our API if you'd like, though on mainnet we suggest checking the hash of the payload against a known constant or something similar.

It'll also tell you UTXO refs you can use for the script bytes etc.

The idea is that as we add more pool types, or deploy to other environments, it's fairly easy to update.